linux_os/guide/services/ssh/ssh_server/disable_host_auth/oval/shared.xml

```diff
@@ -7,32 +8,21 @@
       </affected>
       <description>SSH host-based authentication should be disabled.</description>
     </metadata>
-    <criteria comment="SSH is configured correctly or is not installed"
-    operator="OR">
-      <criteria comment="sshd is not installed" operator="AND">
-        <extend_definition comment="sshd is not required or requirement is unset"
-        definition_ref="sshd_not_required_or_unset" />
-        <extend_definition comment="rpm package openssh-server removed"
-        definition_ref="package_openssh-server_removed" />
-      </criteria>
-      <criteria comment="sshd is installed and configured" operator="AND">
-        <extend_definition comment="sshd is required or requirement is unset"
-        definition_ref="sshd_required_or_unset" />
-        <extend_definition comment="rpm package openssh-server installed"
-        definition_ref="package_openssh-server_installed" />
-        <criterion comment="Check HostbasedAuthentication in /etc/ssh/sshd_config"
-        test_ref="test_sshd_hostbasedauthentication" />
-      </criteria>
```

Comment:
@shawndwells Are you ok if a profile where `sshd` is "required" returns `notapplicable` instead of fail?

Read required as in https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/sshd_required.var.

Current profiles that **require** `sshd` to be installed are:
```console
$ grep -R "sshd_required" */profiles/*
ol7/profiles/stig-ol7-disa.profile:    - sshd_required=yes
rhel7/profiles/ospp.profile:    - sshd_required=yes
rhel7/profiles/rhelh-stig.profile:    - sshd_required=yes
rhel7/profiles/stig-rhel7-disa.profile:    - sshd_required=yes
rhv4/profiles/rhvh-stig.profile:    - sshd_required=yes
```

I'd love to get rid of `sshd_required_or_unset` and `sshd_not_required_or_unset`.

Reply:
If ssh server isn't installed, the checks should be ``not applicable``.

IMHO our role isn't to determine if SSH is required. More that if SSH server is installed, it should be configured a certain way.

Updated profiles to remove sshd_required variable.